client/state/ui/editor/image-editor/actions.js

```diff
@@ -14,16 +14,20 @@ import {
 } from 'state/action-types';
 
 // Doesn't reset image file info (src, fileName, etc).
-export function resetImageEditorState() {
+// additionalData can contain arbitrarily needed data.
```

Comment:
What I meant is that we should document what the reducer actually reads, in this case additonalData.aspectRatio and what this does when we set it